pkgs/misc/tmux-plugins/default.nix

```diff
@@ -127,6 +127,24 @@ in rec {
     };
   };
 
+  dracula = mkDerivation rec {
+    pluginName = "dracula";
+    version = "1.0.1";
+    src = fetchFromGitHub {
+      owner = "dracula";
+      repo = "tmux";
+      rev = "cb1d420a6267c600a8ef0e7a3c00e2474b30ae99";
```

Comment:
But this will. Please either tag the releases in your repo or switch the version to ``unstable-year-month-date``.

Reply:
Done. :+1: